apps/docs-app/app/components/theme-page.gts

```diff
@@ -0,0 +1,257 @@
+import { array } from '@ember/helper';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import Button from '@nrg-ui/core/components/button';
+import Footer from '@nrg-ui/core/components/footer';
+import Form from '@nrg-ui/core/components/form';
+import Icon from '@nrg-ui/core/components/icon';
+import CardContainer from '@nrg-ui/core/components/mktg/card-container';
+import Header from '@nrg-ui/core/components/mktg/header';
+import SectionHeader from '@nrg-ui/core/components/mktg/section-header';
+import bind from '@nrg-ui/core/helpers/bind';
+import { tracked as autoTrack } from 'tracked-built-ins';
+
+export default class extends Component {
+  @tracked
+  searchString = '';
+
+  @tracked
+  customerClass = 'residential';
+
+  @tracked
+  number;
+
+  @tracked
+  text = '';
+
+  @tracked
+  textArea = '';
+
+  @tracked
+  select = '';
+
+  @tracked
+  phone = '';
+
+  @tracked
+  radio = '';
+
+  @tracked
+  checkbox;
+
+  @tracked
+  checkboxGroup = autoTrack(new Array(3));
```

Comment:
Can this just be specified as a tracked array? The import would be simpler.
```suggestion
  checkboxGroup = new TrackedArray(3);
```

Reply:
Tried that. Got the following error message:
![image](https://github.com/user-attachments/assets/3f516a76-9d70-44b1-8b28-ccc12ff124dd)

Seems using the `autoTrack()` method is how we got around this problem in the forms demo page.